packages/next/types/index.d.ts

```diff
@@ -63,6 +63,12 @@ export type NextPage<P = {}, IP = P> = NextComponentType<NextPageContext, IP, P>
 export type PageConfig = {
   amp?: boolean | 'hybrid'
   api?: {
+    /**
+     * Configures or disables body size limit warning. Can take a number or
+     * any string format supported by `bytes`, for example `1000`, `'500kb'` or
+     * `'3mb'`.
+     */
+    responseLimit?: { sizeLimit?: number | string } | boolean
```

Comment:
```suggestion
    responseLimit?: number | string | boolean
```
I don't think we need to nest this under `sizeLimit` as this config is already specific to the size.

Reply:
Is there a reason why `bodyParser` is handled differently? Was using that as a jumping off point.

Reply:
Made the requested change.

Reply:
> Is there a reason why `bodyParser` is handled differently? Was using that as a jumping off point.

Because it's not `bodyParserLimit`. (nor should it be) But I like your logic. 😊